### PR TITLE
Header Logo, Utility, and Main Navs

### PR DESCRIPTION
**Pull request by jpickel**

### CODE REVIEW

**Comment by stevenrayesky** [comment]
Everything is in except the animation for the primary nav.

File: wp-content/themes/avenu/sass/partials/_navs.scss [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  &.tax-payer-portal a {
+    color: $white;
+    background-color: $navy;
+  }
+}
+
+.navigation {
+  &--primary {
+    display: none;
+    @media all and (min-width: 767px) {
```
Comment (jpickel) [inline wp-content/themes/avenu/sass/partials/_navs.scss]: @media queries can use these declarations instead (keeps them in line with the container variables set in bootstrap)

@media (max-width: #{$screen-sm-max}) { }

the variables then are 

$screen-xs-max
$screen-sm-min
$screen-sm-max
$screen-md-min
$screen-md-max
$screen-lg-min

Comment (jpickel) [inline wp-content/themes/avenu/sass/partials/_navs.scss]: @media queries can use these declarations instead (keeps them in line with the container variables set in bootstrap)

@media (max-width: #{$screen-sm-max}) { }

the variables then are

$screen-xs-max
$screen-sm-min
$screen-sm-max
$screen-md-min
$screen-md-max
$screen-lg-min

File: wp-content/themes/avenu/sass/partials/_navs.scss [hunk 2]
```diff
@@ -0,0 +1,103 @@
+.menu-item {
```
Comment (jpickel) [inline wp-content/themes/avenu/sass/partials/_navs.scss]: Make sure these style docs are indented properly... should be 4 spaces or a tab

File: wp-content/themes/avenu/sass/partials/_navs.scss [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+       padding-right: 75px;
+     }
+    @media all and (min-width: 767px) {
+       display: block;
+     }
+  }
+}
+
+#menu-utility-navigation {
+  padding: 10px 0px 15px;
```
Comment (jpickel) [inline wp-content/themes/avenu/sass/partials/_navs.scss]: Use the padding variables set in /modules/elements, when applicable. If one is not aligned with a variable, that's fine, although sometimes you can do math (i.e., 45px can be '($padding-default * 3)' ). 

When something needs to be absolute, and not tied to the variables, it's okay to set a specific number

File: wp-content/themes/avenu/sass/partials/_navs.scss [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  float: right;
+  .menu-item {
+    background-color: $navy;
+    border-radius: 5px;
+    font-size: $font-size-default;
+  }
+  a {
+    color: $white;
+    padding: 10px 25px;
+    letter-spacing: 0.2em;
```
Comment (jpickel) [inline wp-content/themes/avenu/sass/partials/_navs.scss]: I think we need to apply a generic letter spacing first to the body tag in main.scss. Not sure if it's .2em or .5em. Then we can add individual letter spacing when needed.

File: wp-content/themes/avenu/sass/partials/_navs.scss [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+}
+
+.navbar-toggle {
+  position: absolute;
+  right: 0px;
+  top: 50px;
+}
+
+#navbar-collapse-mobile {
+  margin: 0px -45px 0px -15px;
```
Comment (jpickel) [inline wp-content/themes/avenu/sass/partials/_navs.scss]: You can also use the variables here, and add a - in front

File: wp-content/themes/avenu/functions.php [hunk 6]
```diff
@@ -52,5 +55,17 @@ function custom_login_logo() {
 }
 add_action('login_head', 'custom_login_logo');
 
+function wpdocs_theme_name_scripts() {
+    wp_register_script( 'jq', 'https://code.jquery.com/jquery-3.2.1.slim.min.js', array( ), NULL, false );
```
Comment (jpickel) [inline wp-content/themes/avenu/functions.php]: This method is fine; In some other pull requests of mine, you'll want to note that I should move my JS includes from the meta.php file into this file. Just an FYI for when you're reviewing my code.

File: wp-content/themes/avenu/functions.php [hunk 7]
```diff
@@ -25,12 +26,14 @@ function magazino_widgets_init() {
...
 add_theme_support( 'menus' );
 
-register_nav_menus( array(  
-    'primary' => __( 'Primary Navigation' ), 
-    'footer' => __( 'Footer Navigation' ), 
+register_nav_menus( array(
+    'primary' => __( 'Primary Navigation' ),
+    'utility' => __( 'Utility Navigation' ),
+    'footer' => __( 'Footer Navigation' ),
+    'mobile' => __( 'Mobile Navigation' ),
```
Comment (jpickel) [inline wp-content/themes/avenu/functions.php]: Not sure we need another menu position and menu class for styling... shouldn't we just utilize the default bootstrap behavior and use @media queries to add styling that's unique to smaller devices?

**Comment by stevenrayesky** [comment]
@jpickel -

Went through and made the edits. Definitely makes sense to only have the one nav, got rid of the mobile nav. Added a letter-spacing variable to the sass config (what we had on prolease was 0.025em) and applied it to the body tag. Swapped out the nums for variables.

**Pull request merged**